src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -99,24 +100,24 @@ public void storeUser(User user) {
     userEntity.setProperty("aboutMe", user.getAboutMe());
     datastore.put(userEntity);
   }
- 
+
   /**
     * Returns the User owned by the email address, or
     * null if no matching User was found.
     */
   public User getUser(String email) {
- 
+
```

Comment:
You can delete this blank line too so that there's no empty space at the very start of the method.